### PR TITLE
fix: Hotfix v6.1.0

### PR DESCRIPTION
**Pull request by brewmaster012**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes: <PD-XXXX>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Review by ws4charlie — DISMISSED** [review]
looks good

File: x/crosschain/module.go [hunk 1]
```diff
@@ -174,7 +174,11 @@ func (am AppModule) ExportGenesis(ctx sdk.Context, cdc codec.JSONCodec) json.Raw
 func (AppModule) ConsensusVersion() uint64 { return 1 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the capability module.
-func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
+func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
+	if ctx.BlockHeader().Height == 946264 {
+		panic("pause at 946264; please revert back to v6.0.0 and restart")
```
Comment (kingpinXD) [inline x/crosschain/module.go:179]: NIT change this log to use v6.2.0 instead ?

**Review by ws4charlie — APPROVED** [review]

**Review by lumtis — APPROVED** [review]
LGTM
Agree with @kingpinXD comment

**Comment by lumtis** [comment]
It can be closed like https://github.com/zeta-chain/node/pull/859, right?

**Pull request closed without merging**